client/Components/Dashboard/NodeView/CPUusage.jsx

```diff
@@ -1,21 +1,30 @@
 import React, { useEffect, useState } from "react";
 import AnimatedNumber from "animated-number-react";
+import { StaticRouter } from "react-router";
```

Comment:
Is this going to be used anywhere? I see it imported but not sure I understand why. Thinking it may have been an accidental vscode auto import.